tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -182,7 +192,12 @@ protected void initConfigs(String[] args) throws Exception {
     Options opts = new Options();
     opts.addOption("am_address", true, "The address to the application master.");
     CommandLine cliParser = new GnuParser().parse(opts, args);
-    amAddress = cliParser.getOptionValue("am_address", "");
+
+    String amAddress = cliParser.getOptionValue("am_address", "");
```

Comment:
can we also move this to be an environment variable to get rid of all argument passing ?

Reply:
Good idea. Changed to environment variables `AM_HOST` and `AM_PORT` in my latest revision.